src/app/index.tsx

```diff
@@ -52,6 +52,7 @@ export { Application, ApplicationExtensionContext } from "./application";
 export { addSVGIcon, getSVGIcon } from "./resources";
 export { ExportTemplateTarget } from "./template";
 export { Widgets } from "./views/panels/index";
+export * as Buttons from "./components/buttons";
```

Comment:
Optionally, we could add these buttons to the Widgets namespace above.

Reply:
We have exposed widgets from https://github.com/microsoft/charticulator/tree/master/src/app/views/panels/widgets/controls
where we have a button.

Why we need the button from components folder? After  replacing components by Fluent UI components, we will not need those components.

Reply:
That Button does not have a `disabled` property. I will see if it can be added.